### PR TITLE
Fixes aliens being unable to stun

### LINKED ISSUE #31488: Xenomorph Disarms not stunning
Issue reported from Round ID: 77191 (/tg/Station Bagil [ENGLISH] [100% FREE LAG])

As per the title, xenomorph stuns/knockdowns were not working this round. I personally disarmed someone two or three times as a sentinel and they were not stunned at all. I also witnessed a xeno queen try and disarm someone, with no effect. They weren't on harm intent, since they weren't doing any damage.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #31488 
Closes #31490

lmao that round was fun

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -256,7 +256,8 @@
 			apply_damage(damage, BRUTE, affecting, armor_block)
 
 		if(M.a_intent == INTENT_DISARM) //Always drop item in hand, if no item, get stun instead.
-			if(dropItemToGround(get_active_held_item()))
+			var/obj/item/I = get_active_held_item()
+			if(istype(I) && dropItemToGround(I))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/human_defense.dm]: if(I && dropItemToGround(I))

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by ShizCalev** [comment]
Fixes #31488

**Comment by ShizCalev** [comment]
Replicate it here too please;
https://github.com/kevinz000/-tg-station/blob/9f12b782f7464a37ae1401a79f45739f798983fc/code/modules/mob/living/carbon/human/human_defense.dm#L207

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Do what Shiz said

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**